src/shared/components/date-bubble/index.js

```diff
@@ -1,27 +1,27 @@
 // Displays date bubble
+/* eslint-disable max-len */
 
 import React, { Component } from 'react';
 import styles from './style.css';
 
 export default class DateBubble extends Component {
   static propTypes = {
-    date: React.PropTypes.string,
-    month: React.PropTypes.string,
-    year: React.PropTypes.string,
+    startDate: React.PropTypes.string,
```

Comment:
Instead of [`badger-brain` prescribing date formats to your consumers](https://github.com/redbadger/badger-brain/issues/30), why not let your consumers have control over how they want to format dates? Ultimately, I think date formatting is presentational logic that should be fulfilled in the component layer instead. `badger-brain` should only store the an ISO-formatted date string, which can be shaped however your consumers want.